internal/pkg/scaffold/helm/chart.go

```diff
@@ -265,3 +263,19 @@ func createChartFromRemote(destDir string, opts CreateChartOptions) (*chart.Char
 
 	return createChartFromDisk(destDir, chartArchive, false)
 }
+
+func fetchChartDependencies(chartPath string) error {
+	helmHome, ok := os.LookupEnv(environment.HomeEnvVar)
+	if !ok {
+		helmHome = environment.DefaultHelmHome
+	}
+	getters := getter.All(environment.EnvSettings{})
+
+	man := &downloader.Manager{
+		Out:       ioutil.Discard,
```

Comment:
Would printing output help debug issues? Or is it too verbose?

Reply:
`man.Build()` returns errors with descriptive text if things go wrong, and most of the output written to `man.Out` is status about what's happening along the happy path. There are a few error-related things that get printed to though, so I'll update this to print the extra output when `err != nil`